packages/kbn-es-query/src/kuery/node_types/wildcard.js

```diff
@@ -28,7 +28,7 @@ function escapeRegExp(string) {
 
 // See https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#_reserved_characters
 function escapeQueryString(string) {
-  return string.replace(/[+-=&|><!(){}[\]^"~*?:\\/]/g, '\\$&'); // $& means the whole matched string
+  return string.replace(/[+-=&|><!(){}[\]^"~*?:/]/g, '\\$&'); // $& means the whole matched string
```

Comment:
In order to still be able to search for things with backslashes (e.g. `foo.keyword: *\\*`, which is basically "anything that contains a backslash"), we need to change this line:

```suggestion
  return string.replace(/\\(?! )/g, '\\\\') // Only escape backslashes not followed by a space
    .replace(/[+-=&|><!(){}[\]^"~*?:/]/g, '\\$&'); // $& means the whole matched string
```